src/api/content-rest-api/api/categories.api.ts

```diff
@@ -0,0 +1,496 @@
+/*!
```

Comment:
Michal, what with unit tests for that? If you are not going to do that as part of that PR then can you raise jira?

Reply:
I will create Jira for this and assign myself to it, I will add tests after Category tree view not to block work related to categories, is it fine? Or should I add tests in this PR?

Reply:
Fine for me. I just left comment to fix it here or to raise jira for that just to don't forget about it :) Thanks